src/models/generated/ESAC6A.jl

```diff
@@ -43,7 +43,7 @@ Parameters of IEEE Std 421.5 Type AC6A Excitacion System. ESAC6A in PSSE and PSL
 - `Tc::Float64`: Regulator numerator (lead) time constant in s, validation range: `(0, 20)`, action if invalid: `warn`
 - `Va_lim::NamedTuple{(:min, :max), Tuple{Float64, Float64}}`: Limits for regulator output `(Va_min, Va_max)`
 - `Vr_lim::NamedTuple{(:min, :max), Tuple{Float64, Float64}}`: Limits for exciter field voltage `(Vr_min, Vr_max)`
-- `Te::Float64`: Exciter field time constant, validation range: `("eps()", 2.0)`, action if invalid: `error`
+- `Te::Float64`: Exciter field time constant, validation range: `("eps()", 2)`, action if invalid: `error`
```

Comment:
There are many unintentional changes of floats to integers in the auto-generated code.  I'll look into this.

Reply:
@jd-lara JSON3 is the cause of this.  Let me know if this is problematic for you.  We could switch back to JSON with minimal effort.  The only reason I chose JSON3 is that it was up to 4 times faster at reading our data.  We could use JSON for reading our struct descriptors and JSON3 for everything else.  The cost is an extra project dependency.  I will note that there is no correctness issue here.

These examples illustrate the problem:
```
julia> JSON3.read(JSON3.write(Dict("a" => 1.0)), Dict)
Dict{String,Any} with 1 entry:
  "a" => 1
```
```
julia> JSON.parse(JSON.json(Dict("a" => 1.0)))
Dict{String,Any} with 1 entry:
  "a" => 1.0
```

Reply:
I don't think this is a big deal, Julia handles the conversion. It's better to keep the speed. We can switch back to JSON.jl whenever they decide to release 1.0.